Added Pause Menu options to spec Sheet (#164)
Sean-Nishi/Reluctant-Hero: docs/specSheet.md

The Boredom Meter is a countdown to the player character sleeping out of anger. When it reaches 0, the game is over.
Each level has a set timer that ticks up to the moment the player character goes to sleep from boredom.

## Pause Menu
### Menu Options

> The pause menu will have the following options. Resume Game, Restart Level, Settings, and Quit Game.

Reviewer: ```Suggestion
The pause menu will have the following options. [Resume Game](#resume-game), [Restart Level](#restart-level), [Settings](#settings), and [Quit Game](#quit-game).
```

#### Resume Game

> * When selected pause menu will be destroyed and gameplay resumes.

Reviewer: ```suggestion
* When selected, the pause menu is destroyed and gameplay resumes.
```

#### Restart Level

> * When selected current level will be restarted. No progress will be saved.

Reviewer: ```Suggestion
* When selected, the current level is restarted. No progress is saved.
```

#### Settings
* When selected a submenu will open with General, Audio, and Video setting options.
  * General Settings
    * Menu to change general settings of the game.
  * Audio Settings
    * Menu to change audio settings. Includes music volume, effects volume, and menu music.
  * Video Settings
    * Menu to change video settings. Includes brightness, window mode, aspect ratio, and resolution settings.

#### Quit Game

> * When selected end current game and return to main menu.

Reviewer: ```Suggestion
* When selected end the current game and return to the main menu.
```
